package-lock.json

```diff
@@ -9,7 +9,7 @@
       "integrity": "sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g==",
       "dev": true,
       "requires": {
-        "@babel/highlight": "7.8.3"
```

Comment:
It's odd that the `package-lock.json` always seems to be missing most of the version range modifiers (I always get the same thing on my computer)... @BrightnessTM any idea what's causing it (and is it intended for some reason)?

Reply:
(Sorry my review doesn't show more lines, but take a look at all the changes in the file)